### PR TITLE
User password change

### PR DESCRIPTION
**Pull request by PedroAVJ**
### Pull Request - Issue #133: Add New Template for Password Change

**Changes:**

- Introduce a new template: `change_password.html`.
- Utilize a subclassed `PasswordChange` form and view for custom styling and flash message integration.
- Add a button to the profile template to navigate into the change password route consistent with the style of the other applications.
image.
- Block POST requests if the password is:
  - Shorter than 8 characters.
  - Mismatched against the password confirmation.
  - Entirely numeric.
  - Has a `weak` score when using `zxcvbn`.
- Implement a password strength meter using `zxcvbn` for better UX.
image.
- Add the `zxcvbn` JavaScript library to the `base_app` JS folder, following the naming convention: `<library-name>-<version-number>`. This is done similarly to the other JS libraries used in the project.

### CODE REVIEW

**Review by alex-nork — COMMENTED** [review]
Thanks @PedroAVJ! These changes look good, but I'd like to get another set of 👀 on them. @zimventures please check this out when you get back.

In the meantime it'd be awesome if you could add some tests of this new functionality.

File: chirps/account/templates/account/change_password.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{% extends 'base.html' %}
+
+{% load widget_tweaks %}
```
[the remark this answers is not in the document]
Comment (PedroAVJ) [reply chirps/account/templates/account/change_password.html]: It was from a library that I was trying. Ended up being unnecessary.

File: chirps/chirps/settings.py [hunk 2]
```diff
@@ -55,6 +55,7 @@
     'account',
     'policy',
     'embedding',
+    'widget_tweaks',
```
[the remark this answers is not in the document]
Comment (PedroAVJ) [reply chirps/chirps/settings.py]: Removed now

File: chirps/base_app/static/js/zxcvbn-4.4.2.js
[the remark this answers is not in the document]
Comment (PedroAVJ) [reply chirps/base_app/static/js/zxcvbn-4.4.2.js:1]: They are an exact match now. It ended up being only a newline difference and should behave identically as before

**Comment by PedroAVJ** [comment]
I added some tests for both changing the password and for some of the password validation (new password and password confirmation matching, etc..). Also found a small typo in the README.md file

**Pull request merged**